src/arch/xtensa/configs/rembrandt_defconfig

```diff
@@ -11,7 +11,7 @@ CONFIG_FORMAT_CONVERT_HIFI3=n
 CONFIG_LP_SRAM=n
 CONFIG_HAVE_AGENT=n
 CONFIG_COMP_VOLUME=n
-CONFIG_COMP_BLOB=n
+##CONFIG_COMP_BLOB=n
```

Comment:
A few things:

- the commit messages would benefit from a bit more description on what is the intended change and why it is done (virtual dai is removed)

- keeping bisect working -- I think this breaks bisect as first commit removes definitions that are still used in the codebase. the tree should remain buildable after each commit in the series. this will ensure git bisect will work.

- minor style: the common style is to have space after prefixes in git titles, e.g. "drivers: amd: mux update"